### ISSUE TITLE
[Feature request] add option to build Winding routes

### ISSUE THREAD

**Issue by katerynakocherhina** [osmandapp/Osmand#7049]
The option in the navigation mode for calculating the extra curvy route.

**Comment by hakuchi** [comment]
sorry for my question, but will you use this for motorcycle routing?

Maybe also for car navigation it sounds interesting, when there is a option(low, medium, high) to cange how curvy the route should be.

**Comment by mbirth** [comment]
Instead of re-inventing the wheel, maybe a partnership with the guys from kurviger.de (e.g. @boldtrn) would be an alternative?

**Comment by 4sStylZ** [comment]
Hello there, 

New to Osmand, that's a killer feature that I dream about.
I found this : https://github.com/schmidi2/OsmAnd-Motorbike/blob/master/routing/Motorbike.routing.xml

Is that a way to program winding routes ?

Thank's.

**Comment by vshcherb** [comment]
Winding routes needs to be preprocessed somehow before we can build the routes

**Comment by mbirth** [comment]
Instead of reinventing the wheel, routing via the Kurviger API might also be a viable way: https://kurviger.de/api_use/en

**Comment by lotth** [comment]
If you implement such feature, please don't make it specific to motorcycle profile. Car enthusiasts like curvy roads too.

**Comment by markongh** [comment]
[Road Curvature](https://roadcurvature.com/) is another option that already calculates and displays "curvy" roads.  Perhaps this could simply be added as a Map Source or Overlay option in the Configure Map menu.  While this won't do automatic routing and it would still be up to the user to build their route, implementing this as an overlay would certainly reduce the complexity of implementation to help enthusiasts find curvy roads.
